index.js

```diff
@@ -2,8 +2,12 @@ import 'react-native-console-time-polyfill';
 
 import './app/ReactotronConfig';
 import { AppRegistry } from 'react-native';
+import { Client } from 'bugsnag-react-native';
 import { name as appName, share as shareName } from './app.json';
 
+const bugsnag = new Client('72a0364cb361fc8f0fdc6cbf605f9963');
```

Comment:
API Key.
Is it ok to use it here or should we hide from Github?

Reply:
sensitive info of users can't be accessed using API key so I think it should stay

Reply:
I would move this to some separate config file just to not keep API Keys within source code. Also you should consider if you want to get bug reports from apps that are forks of this app especially when they won't share the sources.